catroid/build.gradle

```diff
@@ -179,10 +179,12 @@ android {
         buildConfigField "boolean", "FEATURE_USERBRICKS_ENABLED", "true"
         buildConfigField "boolean", "FEATURE_WEBREQUEST_BRICK_ENABLED", "true"
         buildConfigField "boolean", "FEATURE_MULTIPLAYER_VARIABLES_ENABLED", "true"
+        buildConfigField "boolean", "FEATURE_TESTBRICK_ENABLED", "true"
         resValue "string", "FEATURE_ARDUINO_PREFERENCES_ENABLED", "false"
         resValue "string", "FEATURE_PHIRO_PREFERENCES_ENABLED", "false"
         resValue "string", "FEATURE_RASPI_PREFERENCES_ENABLED", "false"
         resValue "string", "SNACKBAR_HINTS_ENABLED", "false"
+        resValue "string", "debug", "false"
```

Comment:
Please find a better name like debug mode and write it in upper case like the other values e.g. DEBUG_MODE